arm-web/2016-08-01/WebApps.json

```diff
@@ -10324,9 +9947,6 @@
             "$ref": "#/parameters/subscriptionIdParameter"
           },
           {
-            "$ref": "#/parameters/resourceGroupNameParameter"
-          },
-          {
```

Comment:
Server returns 200 as well. Same for `stop()`, `restart()`, `startSlot()`, `restartSot()`, `stopSlot()`.